src/test/data/JsonSerializableLibTaskTest/typicalPatronsLibTask.json

```diff
@@ -1,5 +1,5 @@
 {
-  "_comment": "AddressBook save file which contains the same Person values as in TypicalPersons#getTypicalAddressBook()",
+  "_comment": "LibTask save file which contains the same Patron and Book values as in TypicalLibTask#getTypicalLibTask()",
```

Comment:
Good update of the comment to reflect ```libtask``` and also updated it to contain patron and book.